config/heightdepvar.go

```diff
@@ -45,14 +45,14 @@ func (hdi *HeightDependentUint256) GetValueAtHeight(height uint32) common.Uint25
 }
 
 type HeightDependentBool struct {
-	heights []uint32
-	values  []bool
+	Heights []uint32
```

Comment:
It's better to change all `HeightDependentXXX` variables to make them consistent